server/player.lua

```diff
@@ -337,6 +337,29 @@ function QBCore.Player.CreatePlayer(PlayerData, Offline)
         return true
     end
 
+    function self.Functions.TransferMoneyTo(emitermoneytype, receivercid, receivermoneytype, quant, reason)
```

Comment:
`quant` should be replaced by `amount` since it's the wording use by default for this kind of property